src/middleware.js

```diff
@@ -18,13 +18,23 @@ function bindInterceptors(client, getState, middlewareInterceptors = {}, clientI
   });
 }
 
+const requests = {};
+
 export const multiClientMiddleware = (clients, customMiddlewareOptions) => {
   const middlewareOptions = { ...defaultOptions, ...customMiddlewareOptions };
   const setupedClients = {};
+  let actionRequestId = null;
   return ({ getState, dispatch }) => next => action => {
     if (!middlewareOptions.isAxiosRequest(action)) {
       return next(action);
     }
+
+    if (action.payload.cancellable) {
+      const requestId = typeof requests[action.type] !== 'undefined' ? requests[action.type] + 1 : 1;
```

Comment:
You'll probably need to move this after the `getActionTypes()` call below and use the `REQUEST` identifier as the key, because not all actions have a singular `type` property.